bin/migrate-oats-data/common/constants.py

```diff
@@ -1,3 +1,4 @@
 OATS_ETL_USER = "oats_etl"
+OATS_ETL_POST_LAUNCH_USER = "oats_etl_post_launch"
```

Comment:
I do not think we need a "oats_etl_post_launch" user. All ETL data will be treated the same no matter what user was used to import it

Reply:
set back to oats_etl_user